package-lock.json

```diff
@@ -1664,8 +1664,7 @@
               "version": "2.1.1",
               "resolved": false,
               "integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8=",
-              "dev": true,
-              "optional": true
+              "dev": true
```

Comment:
What are the package.lock changes here?

Reply:
This is a copy of the `package-lock.json` file from the upstream repo `Can-Sahin/react-boilerplate-typescript`. If you feel like you need to tweak that file's contents, you are welcome to do so.

Reply:
@cdoremus-vfc It looks like you made changes to this file. I'm wondering why and what the changes you made a are for.

Reply:
I didn't purposely make changes, but my experience is that `package-lock.json` can 'magically' change without my input. It might be my version of `npm` has this magical powers (6.4.1).

Anyway, I'll get another copy from the upstream repo and push it to this branch.

Reply:
You could probably also just revert the last commit.

Reply:
Copied in upstream version in f454bc2c